### PR TITLE
Capstone skeleton

### PR DESCRIPTION
**Pull request by AzizRAA**
This project is about CyberXlab summer course.
built with HTML, CSS, and JAVASCRIPT.
We have two pages the main page which is the home page and the about page which is the second page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Capstone 1
 
 > Description the project.
```
Comment (emmanuellekamwa) [inline README.md:6]: - [ ] Please use this section to add a description of your project. It makes your readme look more professional.

File: README.md [hunk 2]
```diff
@@ -36,22 +36,6 @@ To get a local copy up and running follow these simple example steps.
 
 ### Deployment
 
-
-
```
Comment (emmanuellekamwa) [inline README.md:40]: - [ ] Please delete this section since it is empty.

File: README.md [hunk 3]
```diff
@@ -66,7 +50,6 @@ Give a ⭐️ if you like this project!
 
 - Hat tip to anyone whose code was used
 - Inspiration
-- etc
```
Comment (emmanuellekamwa) [inline README.md:69]: - [ ] On the acknowledgment section you have to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ ] I have noticed that you did not record a presentation video. Please you have to do so.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
-- Frameworks
-- Technologies used
+- Javascript , CSS , HTML5 
+- Bootstrap
+- VSCode , Linters , Github Deploy
 
-## Live Demo (if available)
+## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```
Comment (emmanuellekamwa) [inline README.md:17]: - [ ] Your live demo link is not working. Please try and fix that

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🍾✔🎉**
Hello @AzizRAA 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AzizRAA** [comment]
https://www.loom.com/share/5341b60cc36f497683a7774fbdd84127
Loom

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @AzizRAA 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**